controllers/util/mw_util.go

```diff
@@ -327,7 +334,80 @@ func (mwu *MWUtil) generateVRGClusterRoleBindingManifest() (*ocmworkv1.Manifest,
 			Kind:     "ClusterRole",
 			Name:     "open-cluster-management:klusterlet-work-sa:agent:volrepgroup-edit",
 		},
-	})
+	}
+
+	olmClusterRole = &rbacv1.ClusterRole{
+		TypeMeta:   metav1.TypeMeta{Kind: "ClusterRole", APIVersion: "rbac.authorization.k8s.io/v1"},
+		ObjectMeta: metav1.ObjectMeta{Name: "open-cluster-management:klusterlet-work-sa:agent:olm-edit"},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{"operators.coreos.com"},
+				Resources: []string{"operatorgroups"},
+				Verbs:     []string{"create", "get", "list", "update", "delete"},
+			},
+			{
+				APIGroups: []string{"operators.coreos.com"},
+				Resources: []string{"catalogsources"},
+				Verbs:     []string{"create", "get", "list", "update", "delete"},
+			},
+		},
+	}
+)
+
+func olmRoleBinding(namespaceName string) *rbacv1.RoleBinding {
+	return &rbacv1.RoleBinding{
+		TypeMeta: metav1.TypeMeta{Kind: "RoleBinding", APIVersion: "rbac.authorization.k8s.io/v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "open-cluster-management:klusterlet-work-sa:agent:olm-edit",
+			Namespace: namespaceName,
+		},
+		Subjects: []rbacv1.Subject{
+			{
+				Kind:      "ServiceAccount",
+				Name:      "klusterlet-work-sa",
+				Namespace: "open-cluster-management-agent",
+			},
+		},
+		RoleRef: rbacv1.RoleRef{
+			APIGroup: "rbac.authorization.k8s.io",
+			Kind:     "ClusterRole",
+			Name:     "open-cluster-management:klusterlet-work-sa:agent:olm-edit",
+		},
+	}
+}
+
+func operatorGroup(namespaceName string) *operatorsv1.OperatorGroup {
```

Comment:
(not related to this PR) We need to rethink our CSV `installModes` to a SingleNamespace and hence add `spec.targetNamespaces` here. This is based on the understanding [here](https://olm.operatorframework.io/docs/concepts/crds/operatorgroup/#installmodes-and-supported-operatorgroups)

Reply:
Ok.

Reply:
An issue is needed to track this.

Reply:
Issue https://github.com/RamenDR/ramen/issues/363 tracks this issue.